### DISCUSSION TITLE
deleted forked repo

### DISCUSSION THREAD

**Discussion by AdMatay** [community/community#57868] [Enterprise]
### Select Topic Area

Question

### Body

Hello, 
I'm pretty new to github, and I have made a mistake and accidently deleted a forked repo. now I can't stress this enough but it is super important that I restore that repo. I have made an attempt at contacting git but can't seem to get any help. I'm considering buying git's enterprise plan just to contact them. Is this a good idea, given my situation?

**Comment by radar-blip** [comment]
If you go to your settings, then to repositories, you may be able to un-delete the repository from there, under `Deleted repositories`.

**Reply by AdMatay** [reply]
I tried that, because it is a forked repository then github is unwilling to return it

**Reply by fykaa** [reply]
Were you able to solve this @AdMatay ?

**Comment by phuccoder2000** [comment]


**Reply by fykaa** [reply]
How much time does GitHub Support takes to reply back to the ticket
